### DISCUSSION TITLE
Search by SKU

### DISCUSSION THREAD

**Discussion by rsidhaarth** [Blair2004/NexoPOS#447] [Ideas]
Hi Team,

I have already raised the request for Live Search by product name. And I can see it is now available for Bar Code, it will be better if you add for SKU as well.

![image](https://user-images.githubusercontent.com/16856248/147819095-1c010695-74b5-4012-ac62-310ea13fc930.png)

and so, the cashier will enter the SKU or Barcode number it gets added to cart

**Comment by Blair2004** [comment]
Click the lens button to search product using their name or SKU.
